### PR TITLE
Remove pause reservations from user profile

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFLSBP-424

#### Description
This PR removes the "pause your reservations" section from the patron page. 
Changes to spacing between sections have also been made to reflect Figma design.

#### Screenshot of the result
-

#### Additional comments or questions
-

### CODE REVIEW

**Review by spaceo — COMMENTED** [review]
LGTM 👍

File: src/apps/patron-page/PatronPage.tsx [hunk 1]
```diff
@@ -110,61 +107,56 @@ const PatronPage: FC = () => {
...
+            id="delete-patron-link"
+            href={deletePatronUrl}
+            className="link-tag"
+          >
+            {t("patronPageDeleteProfileLinkText")}
+          </Link>
         </div>
-      </form>
-      {patron && (
-        <PauseReservation user={patron} id={pauseReservation as string} />
```
Comment (spaceo) [inline src/apps/patron-page/PatronPage.tsx:165]: What about the `PauseReservation` component? Shouldn't it be removed?

**Review by spaceo — CHANGES_REQUESTED** [review]
Looks goodie. One question though.

File: src/apps/patron-page/PatronPage.tsx (hunk 1 above)
Comment (Adamik10) [reply src/apps/patron-page/PatronPage.tsx:165]: Great question! Thanks for asking 🥇 
I checked though, and it is still in use in the reservation list component:
image

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**